src/main/webapp/feedback/post-icon-feedback.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Stores icon feedback data in the database. */
+export default class PostIconFeedback {
+  static #ENDPOINT_FEEDBACK = '/icon-feedback';
+  static #PARAM_LECTURE_ID = 'lectureId';
```

Comment:
nit: is it possible to make these static and const?

Reply:
No in javascript classes